src/templates/career-details.tsx

```diff
@@ -45,6 +46,8 @@ const CareerPage = (props: CareerPageProps): JSX.Element => {
         position={position}
         complementPath={pageContext.translation}
       />
+
+      <CareerDetailsStructuredData position={position} />
```

Comment:
Please put it on top, right below `SEO`. It modifies the HTML, which is also on top of the file.

Reply:
I changed the order for @milljoniaer